packages/core/useFetch/index.ts

```diff
@@ -0,0 +1,161 @@
+import { Ref, ref, unref, watch, isRef, computed, ComputedRef } from 'vue-demi'
+import { Fn, MaybeRef } from '@vueuse/shared'
+
+interface UseFetchReturn {
+  /**
+   * Indicates if the fetch request has finished
+   */
+  isFinished: Ref<boolean>
+
+  /**
+   * The statusCode of the HTTP fetch response
+   */
+  status: Ref<number | null>
```

Comment:
```suggestion
  statusCode: Ref<number | null>
```

Reply:
Oh right I should have renamed that, fixed now